buruberi-core/src/main/java/is/hello/buruberi/bluetooth/stacks/android/DeviceSupport.java

```diff
@@ -29,6 +29,50 @@
     static final String MANUFACTURER_MOTOROLA = "motorola";
 
     static boolean isModelSupported(@NonNull String manufacturerPattern, @NonNull String model) {
+        switch(model){
+            case "Nexus 5X":
+            case "Nexus 6P":
+            case "HTC6525LVW":
+            case "SM-G800F":
+            case "SM-N9005":
+            case "GT-I9195":
+            case "D5503":
+            case "LG-D855":
+            case "D6603":
+            case "C6903":
+            case "E5823":
+            case "VS986":
+            case "LG-D802":
+            case "ASUS_Z00AD":
+            case "D6503":
+            case "E6653":
+            case "LG-H815":
+            case "ONE A2003":
+            case "831C":
+            case "LG-D851":
+            case "D5833":
+            case "MotoG3":
+            case "HTC6500LVW":
+            case "SCH-I435":
+            case "D6616":
+            case "LG-D850":
+            case "LG-H811":
+            case "LGLS990":
+            case "VS980 4G":
+            case "E6533":
+            case "LG-D852":
+            case "HTC6535LVW":
+            case "Pixel C":
+            case "ONE A2005":
+            case "D6708":
+            case "LG-D405":
+            case "ONE E1003":
+            case "LG-D800":
+            case "One X":
+            case "HTC One M8s":
+                return true;
+        }
```

Comment:
you'll probably want to put this switch statement inside of getDeviceSupportLevel, before the manufacturer specific code